.github/workflows/check-deps.yml

```diff
@@ -0,0 +1,18 @@
+name: Check for dependency changes
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
As I said in my previous comment:
> Could you make this run hourly, but at "hour + 30 minutes", so that there is better chance to not miss an update from Haiti that may have happen at the top of the previous hour.